scanner/src/main/kotlin/storages/ClearlyDefinedStorage.kt

```diff
@@ -122,13 +112,30 @@ class ClearlyDefinedStorage(
                 coordinates.revision.orEmpty()
             )
 
-            val version = findScanCodeVersion(tools, coordinates)
-            if (version != null) {
-                listOfNotNull(loadScanCodeResult(coordinates, version))
-            } else {
-                logger.debug { "$coordinates was not scanned with any version of ScanCode." }
+            val toolsVersionsByName = tools.mapNotNull { it.withoutPrefix("$coordinates/") }
```

Comment:
The plural `tools` in `toolsVersionsByName` looked a bit unusual within our code base IIRC. 
Maybe instead `toolVersionsByName`, `toolVersionsByToolName`, `versionsForToolName` or `versionsForName` ?

Reply:
Yeah, the plural "tools" in there was a typo, so I went for "toolVersionsByName".